Examples/Tests/moving_window_boost/moving_window_v_boost.py

```diff
@@ -0,0 +1,22 @@
+import os
+import yt
```

Comment:
Could add a few lines of comments here, in order to explain what the tests does, and what it is checking at the end of the simulation?